config/action.d/iptables-allports.conf

```diff
@@ -21,6 +23,10 @@ actionstart = <iptables> -N f2b-<name>
               <iptables> -A f2b-<name> -j <returntype>
               <iptables> -I <chain> -p <protocol> -j f2b-<name>
 
+actionstart6 = <ip6tables> -N f2b-<name>6
+               <ip6tables> -A f2b-<name>6 -j <returntype>
+               <ip6tables> -I <chain> -p <protocol> -j f2b-<name>6
```

Comment:
I think just to be more obvious.
Here and there you have to care the naming, e.g. when using *ipset*. 
I don't have a strong opinion on this. If you like, I can omit the `6' at the end where it isn't necessary to be distinguished.